src/functions-reference/bounded_discrete_distributions.Rmd

```diff
@@ -155,6 +155,132 @@ The log binomial probability mass of n successes in N trials given
 logit-scaled chance of success alpha dropping constant additive terms
 `r since("2.25")`
 
+## Binomial-logit generalized linear model (Logistic Regression) {#binomial-logit-glm}
+
+Stan also supplies a single function for a generalized linear model
+with Binomial likelihood and logit link function, i.e. a function
+for logistic regression with aggregated outcomes. This provides a more efficient
+implementation of logistic regression than a manually written
+regression in terms of a Binomial likelihood and matrix
+multiplication.
+
+### Probability mass function
+
+Suppose $N \in \mathbb{N}$, $x\in \mathbb{R}^{n\cdot m}, \alpha \in \mathbb{R}^n, \beta \in \mathbb{R}^m$, and $n \in
+\{0,\ldots,N\}$.  Then \begin{align*}
+&\text{BinomialLogitGLM}(n~|~N, x, \alpha, \beta) = \text{Binomial}(n~|~N,\text{logit}^{-1}(\alpha_i + x_i\cdot
+\beta)) \\ &= \binom{N}{n} \left( \text{logit}^{-1}(\alpha_i + \sum_{1\leq j\leq m}x_{ij}\cdot \beta_j) \right)^{n}  \left( 1 -
+\text{logit}^{-1}(\alpha_i + \sum_{1\leq j\leq m}x_{ij}\cdot \beta_j) \right)^{N - n}.  \end{align*}
+
+### Sampling statement
+
+`n ~ ` **`binomial_logit_glm`**`(N, x, alpha, beta)`
+
+Increment target log probability density with `binomial_logit_glm_lupmf(n | N, x, alpha, beta)`.
+`r since("2.34")`
+
+<!-- real; binomial_logit_glm ~; -->
+\index{{\tt \bfseries binomial\_logit\_glm }!sampling statement|hyperpage}
+
+### Stan Functions
+
+<!-- real; binomial_logit_glm_lpmf; (int n | int N, matrix x, real alpha, vector beta); -->
+\index{{\tt \bfseries binomial\_logit\_glm\_lpmf  }!{\tt (int n \textbar\ int N, matrix x, real alpha, vector beta): real}|hyperpage}
+
+`real` **`binomial_logit_glm_lpmf`**`(int n | int N, matrix x, real alpha, vector beta)`<br>\newline
```

Comment:
Like our other function doc, this should name the arguments.  alpha is an intercept, beta is a vector slopes, x is the data matrix, N is the total count and n is the count of successes.  

Now if `x` is a matrix, doesn't `n` and `N` have to be 1D arrays?

Reply:
> Like our other function doc, this should name the arguments. alpha is an intercept, beta is a vector slopes, x is the data matrix, N is the total count and n is the count of successes.

That would be inconsistent with the current doc for other `_glm` functions:

 - [`normal_id_glm`](https://github.com/stan-dev/docs/blob/0398ca443f4ad9bf98560a50ac0f6615b0438c84/src/functions-reference/unbounded_continuous_distributions.Rmd#L235)
 - [`bernoulli_logit_glm`](https://github.com/stan-dev/docs/blob/0398ca443f4ad9bf98560a50ac0f6615b0438c84/src/functions-reference/binary_distributions.Rmd#L181)

> Now if x is a matrix, doesn't n and N have to be 1D arrays?

No, they would be broadcast to match. This is the same way in the [`bernoulli_logit_glm` doc](https://github.com/stan-dev/docs/blob/0398ca443f4ad9bf98560a50ac0f6615b0438c84/src/functions-reference/binary_distributions.Rmd#L181)

Reply:
Then `normal_id_glm` and `bernoulli_logit_glm` should be fixed to match the rest of our doc.  Given that we've thrown out consistency, there's no need for any new GLM to be consistent with the other GLM doc.  I'd rather it be consistent with the rest of our doc as that's where it's going to go in the end.

Reply:
I should've added that this doesn't need to be done as part of this PR.  If you leave the new `_glm` like the other GLM code, I can just fix it in a pass to make everything consistent again.